Permit user with Write or Full permission to delete files and change Document to Files (#66)
Grupo-AFAL/documentation: CHANGELOG.md

# Changelog

All notable changes to this project will be documented in this file.

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

> ## [0.3.0] - 2022-07-29

Reviewer: ```suggestion
## [0.3.0] - 2022-08-03
```

> - Validate documents to eliminate only if user has privileges to delete page (Full privileges)

Reviewer: ```suggestion
- Allow users with `full page privileges` to delete page documents
```
